cmd/api/src/api/v2/audit.go

```diff
@@ -93,23 +94,34 @@ func (s Resources) ListAuditLogs(response http.ResponseWriter, request *http.Req
 		}
 
 		queryParams := request.URL.Query()
+		if queryParams.Has(model.PaginationQueryParameterSkip) {
```

Comment:
A comment here might be helpful since it goes against what our other endpoints do, but is necessary to keep backwards compatibility with the current published API spec